addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -7453,6 +7455,349 @@ registry.SelectTemplate = SnippetOptionWidget.extend({
     },
 });
 
+let dbSocialValues;
```

Comment:
I'll let @qsm-odoo have a first look as he just fine-tuned the social media.

But I am wondering why are you not splitting the options in files?
8.000 lines is a bit much IMHO, making it hard to navigate and find where you want quickly, + starting to make my IDE not happy when loading such files.

Also, if it was in its own file, it could be considered as a move by git/github in a separate commit, if I am not mistaken. Easier to review :grimacing: 

Just my opinion tho (:

Reply:
I'll split the file, you've made a point :+1: